src/main/java/uk/gov/ons/ssdc/caseprocessor/schedule/FulfilmentProcessor.java

```diff
@@ -23,22 +23,22 @@ public FulfilmentProcessor(
 
   @Transactional
   public void addFulfilmentBatchIdAndQuantity() {
-    List<String> fulfilmentCodes = fulfilmentToProcessRepository.findDistinctFulfilmentCode();
+    List<String> packCodes = fulfilmentToProcessRepository.findDistinctPackCode();
 
-    fulfilmentCodes.forEach(
-        fulfilmentCode -> {
+    packCodes.forEach(
+        packCode -> {
           UUID batchId = UUID.randomUUID();
           log.with("batch_id", batchId)
-              .with("fulfilment_code", fulfilmentCode)
+              .with("fulfilment_code", packCode)
```

Comment:
Should this be  .with("pack code", packCode)?

Reply:
fixed